index.html

```diff
@@ -331,17 +447,27 @@ <h4>Skills</h4>
               src="images/bg-image.png"
               alt="contact-image"
             />
+            <img
+              class="desktop-bg-image"
+              src="images/contact-form-desktop-background.png"
+              alt="contact-image"
+            />
           </div>
           <div class="contacts-content">
             <div class="contact-intro">
               <h2>Contact me</h2>
-              <p class="pitch">
+              <p class="pitch mobile-pitch">
                 If you have an application you <br />
                 are interested in developing, a <br />
                 feature that you need built or a <br />
                 project that needs coding, I'd <br />
                 love to help with it
               </p>
+              <p class="desktop-pitch">
+                If you have an application you are interested in developing, a
+                feature that you need built or a project that needs coding, I'd
+                love to help with it
```

Comment:
Mobile Version:

- [ ] Kindly remove the extra spacing according to the Figma design, as shown in the attached loom video.
- [ ] Make sure to give spacing in the card below the heading and list icons as per the video attached.
- [ ] Kindly implement the button interactions for all the buttons in the project. Interactions like hover and pressed.
<img width="463" alt="Screenshot 2022-06-02 at 6 58 56 PM" src="https://user-images.githubusercontent.com/70269507/171676595-378210f5-a06c-469e-a0e5-03e523bd64ca.png">


Desktop Version:

- [ ] Kindly remove the extra spacing as shown in the loom video.
- [ ] Kindly make sure that the images are 50% of the card and bigger in size.